releases/index.md

```diff
@@ -25,6 +25,8 @@ After downloading your desired release, learn how to [install CockroachDB](../{{
   {% assign old_release_format = "True" %}
 {% endif %} {% comment %} For all releases prior to and including 20.1, we use different logic to generate the page (vXX.Y.Z.html vs vXX.Y.html#vXX-Y-Z). {% endcomment %}
 
+{% assign shareleases = "v21.1,v21.2,v22.1" | split: "," %} {% comment %} For all Production releases 21.1 and later, we provide shasum files as well. {% endcomment %}
```

Comment:
I'd suggest doing an inverse search, like what I did for "old_release_format". Otherwise this looks awesome!

Reply:
Thanks! Perfect solution. Also combined the `if ... and` logic in a slightly smarter way. First time writing Jekyll! Let me know what you think.